Assets/Scripts/Models/InputOutput/ModsManager.cs

```diff
@@ -14,13 +14,26 @@ public class ModsManager
 {
     private DirectoryInfo[] mods;
 
-    public ModsManager(string dataPath)
+    public ModsManager()
     {
-        string modsPath = System.IO.Path.Combine(dataPath, "Mods");
-        DirectoryInfo modsDir = new DirectoryInfo(modsPath);
-        mods = modsDir.GetDirectories();
+        mods = GetModsFiles();
+        if (SceneManagerProjectPorcupine.Instance.GetCurrentScene() == SceneManagerProjectPorcupine.MainMenuSceneName)
```

Comment:
It feels like this could be a method `SceneManagerProjectPorcupine.isAtMainMenu()` or similar.